taipy/gui/server.py

```diff
@@ -256,8 +256,8 @@ def _get_async_mode(self) -> str:
 
     def _apply_patch(self):
         if self._get_async_mode() == "gevent" and util.find_spec("gevent"):
-            from gevent import monkey
-
+            from gevent import monkey, get_hub
```

Comment:
```suggestion
            from gevent import get_hub, monkey

```
for the linter (including the empty line)